package.json

```diff
@@ -12,7 +12,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
     "ethereum-waffle": "^3.0.0",
-    "ethers": "^5.0.0",
+    "ethers": "5.0.0",
```

Comment:
Had to downgrade Ethers because the EIP1559 support was bugging out transactions in the E2E tests (Kovan does not support the new tx types yet)